src/services/twilio/sms.ts

```diff
@@ -0,0 +1,34 @@
+import Twilio from "twilio";
+require("dotenv").config();
+
+const accountSid = process.env.ACCOUNT_SID;
+const authToken = process.env.AUTH_TOKEN;
+const twilioPhone = process.env.TWILIO_PHONE;
+const recipientPhoneNumber = process.env.RECEPIENT_PHONE_NUMBER;
+
+const client = Twilio(accountSid, authToken);
+
+export async function sendSMS(body: string) {
+    let to: string;
+
+    if (recipientPhoneNumber === undefined) {
+        console.error(
+            "Recipient phone number is not defined. Error messages will not be sent to admin."
+        );
+        return;
+    }
+
+    to = recipientPhoneNumber;
+    try {
+        const message = await client.messages.create({
+            body,
+            from: twilioPhone,
+            to,
+        });
```

Comment:
@ibrahim681 let me know if you'd like to be added iA

Reply:
that would be great😊